pyvista/jupyter/__init__.py

```diff
@@ -44,6 +58,10 @@ def _validate_jupyter_backend(backend):
             raise ImportError('Please install `pythreejs` to use this feature.')
 
     if backend == 'ipyvtklink':
+        warnings.warn(
+            '`ipyvtklink` backend is deprecated and has been replaced by the `trame` backend.',
+            PyVistaDeprecationWarning,
```

Comment:
```suggestion
            PyVistaDeprecationWarning, stacklevel=3,
```
Might not always be correct (?) but at least for MNE this gets me:
```
/Users/larsoner/python/mne-python/mne/viz/backends/_notebook.py:1458: PyVistaDeprecationWarning: `ipyvtklink` backend is deprecated and has been replaced by the `trame` backend.
  viewer = self.plotter.show(
```
instead of the less informative
```
/Users/larsoner/python/pyvista/pyvista/jupyter/notebook.py:55: PyVistaDeprecationWarning: `ipyvtklink` backend is deprecated and has been replaced by the `trame` backend.
  warnings.warn(
```